### PR TITLE
Updating raven

### PR DESCRIPTION
**Pull request by erinyoung**
Raven can be updated.

I changed the version and added in a test, but it's basically the same dockerfile as before.

Pull Request (PR) checklist:
- [X] Include a description of what is in this pull request in this message.
- [X] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [X] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [X] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [X] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [X] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [X] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [X] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [X] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

File: raven/1.8.1/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    && rm -rf /var/lib/apt/lists/*
+
+# install raven
+RUN wget https://github.com/lbcb-sci/raven/archive/refs/tags/${RAVEN_VERSION}.tar.gz && \
+    tar -xvf ${RAVEN_VERSION}.tar.gz && \
+    rm ${RAVEN_VERSION}.tar.gz 
+  
+WORKDIR raven-${RAVEN_VERSION}
+
+RUN cmake -S ./ -B ./build -DRAVEN_BUILD_EXE=1 -DCMAKE_BUILD_TYPE=Release -Wno-dev && \
```
Comment (kapsakcj) [inline raven/1.8.1/Dockerfile:43]: I think you might be missing the intermediate step `cmake --build build` here? (disclaimer: I have not tested, just looking at the Raven docs)
```
cmake -S ./ -B./build -DRAVEN_BUILD_EXE=1 -DCMAKE_BUILD_TYPE=Release
cmake --build build
cmake --install ./build

```

Comment (erinyoung) [reply raven/1.8.1/Dockerfile:43]: HOW'D YOU GET THAT FROM TEH RAVEN DOCS!?

Comment (erinyoung) [reply raven/1.8.1/Dockerfile:43]: And, yes, that fixed it.

**Comment by erinyoung** [comment]
This is now ready for review. 

Other things that have been updated :
the image is now jammy
added in boost and doxygen to get rid of those error messages

File: raven/1.8.1/Dockerfile (hunk 1 above)
Comment (kapsakcj) [reply raven/1.8.1/Dockerfile:43]: That's what a decade of scanning bioinformatics software documentation will do for ya 😜

**Comment by kapsakcj** [comment]
Everything looks great here, I want to run at least one sample through locally to test functionality. Then I'll approve and merge

File: raven/1.8.1/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+LABEL maintainer="Erin Young"
+LABEL maintainer.email="eriny@utah.gov"
+
+# otherwise tzdata won't install
+ARG DEBIAN_FRONTEND=noninteractive
+
+# install dependencies via apt-get or yum if using a centos or fedora base 
+RUN apt-get update && apt-get install -y \
+    wget \
+    git \
```
Comment (kapsakcj) [inline raven/1.8.1/Dockerfile:25]: is `git` required? Can you remove it if not required

Comment (erinyoung) [reply raven/1.8.1/Dockerfile]: That's a good question. Just a sec.

Comment (erinyoung) [reply raven/1.8.1/Dockerfile]: Yes. It's required:

```
#10 [app 5/6] RUN cmake -S ./ -B ./build -DRAVEN_BUILD_EXE=1 -DCMAKE_BUILD_TYPE=Release -Wno-dev &&     cmake --build build &&     cmake --install ./build
#10 0.152 -- The CXX compiler identification is GNU 11.2.0
#10 0.161 -- Detecting CXX compiler ABI info
#10 0.251 -- Detecting CXX compiler ABI info - done
#10 0.262 -- Check for working CXX compiler: /usr/bin/c++ - skipped
#10 0.263 -- Detecting CXX compile features
#10 0.263 -- Detecting CXX compile features - done
#10 0.301 CMake Error at /usr/share/cmake-3.22/Modules/ExternalProject.cmake:2666 (message):
#10 0.301   error: could not find git for clone of bioparser-populate
#10 0.301 Call Stack (most recent call first):
#10 0.301   /usr/share/cmake-3.22/Modules/ExternalProject.cmake:3716 (_ep_add_download_command)
#10 0.301   CMakeLists.txt:23 (ExternalProject_Add)
```

Comment (kapsakcj) [reply raven/1.8.1/Dockerfile]: ah ok, no worries

**Comment by kapsakcj** [comment]
Docker image builds successfully on my local machine, and I was able to assemble some Nanopore reads for a Chlamydia isolate 👍 

I'm good to approve and merge.

I'll deploy to dockerhub and quay momentarily

**Review by kapsakcj — APPROVED** [review]

**Comment by kapsakcj** [comment]
Just to show that it ran successfully:

```bash
$ raven --polishing-rounds 2 --threads 4 --graphical-fragment-assembly chlamydia.gfa great-dataset-trycycler-chlamydia-trachomatic.reads.fastq.gz > chlamydia.fasta
[raven::] loaded 43662 sequences 4.366431s
[raven::Graph::Construct] minimized 0 - 43662 / 43662 3.729751s
[raven::Graph::Construct] mapped sequences 20.985285s
[raven::Graph::Construct] annotated piles 0.167165s
[raven::Graph::Construct] removed contained sequences 0.098488s
[raven::Graph::Construct] removed chimeric sequences 0.008849s
[raven::Graph::Construct] reached checkpoint 0.063885s
[raven::Graph::Construct] minimized 0 - 250 / 250 0.667523s
[raven::Graph::Construct] mapped valid sequences 0.390320s
[raven::Graph::Construct] updated overlaps 0.000944s
[raven::Graph::Construct] removed false overlaps 0.009353s
[raven::Graph::Construct] stored 446 nodes 0.056295s
[raven::Graph::Construct] stored 2528 edges 0.000000s
[raven::Graph::Construct] reached checkpoint 0.184084s
[raven::Graph::Construct] 26.421541s
[raven::Graph::Assemble] removed transitive edges 0.001717s
[raven::Graph::Assemble] reached checkpoint 0.153306s
[raven::Graph::Assemble] removed tips and bubbles 0.000069s
[raven::Graph::Assemble] reached checkpoint 0.129610s
[raven::Graph::Assemble] removed long edges 0.019848s
[raven::Graph::Assemble] reached checkpoint 0.117899s
[raven::Graph::Assemble] 0.609495s
[racon::Polisher::Polish] minimized 0 - 2 / 2 0.130331s
[racon::Polisher::Polish] mapped sequences 6.410203s
[racon::Polisher::Polish] found 39835 overlaps 6.588082s
[racon::Polisher::Polish] reverse complemented sequences 0.001208s
[racon::Polisher::Polish] aligned 39835 / 39835 overlaps [================] 31.353080s
[racon::Polisher::Polish] prepared 2056 window placeholders 0.379063s
[racon::Polisher::Polish] called consensus for 2056 / 2056 windows [================] 49.748617s
[racon::Polisher::Polish] 88.084009s
[raven::Graph::Polish] reached checkpoint 0.006300s
[racon::Polisher::Polish] minimized 0 - 2 / 2 0.148343s
[racon::Polisher::Polish] mapped sequences 7.386802s
[racon::Polisher::Polish] found 41620 overlaps 7.581897s
[racon::Polisher::Polish] reverse complemented sequences 0.000655s
[racon::Polisher::Polish] aligned 41620 / 41620 overlaps [================] 25.554597s
[racon::Polisher::Polish] prepared 2104 window placeholders 0.380070s
[racon::Polisher::Polish] called consensus for 2104 / 2104 windows [================] 48.809989s
[racon::Polisher::Polish] 82.341551s
[raven::Graph::Polish] reached checkpoint 0.001228s
[raven::] 202.028060s

$ ls -lh
total 209M
-rw-rw-r-- 1 1001 1002 1.7K Aug  3 18:53 Dockerfile
-rw-r--r-- 1 1001 1002 1.1M Aug  3 19:29 chlamydia.fasta
-rw-r--r-- 1 1001 1002 1.1M Aug  3 19:29 chlamydia.gfa
-rw-r--r-- 1 1001 1002 207M Aug  3 18:56 great-dataset-trycycler-chlamydia-trachomatic.reads.fastq.gz
-rw-r--r-- 1 1001 1002 537K Aug  3 19:29 raven.cereal

```

**Comment by kapsakcj** [comment]
@erinyoung I'll let you delete the dev branch if you're done with it

**Pull request merged**